.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish package release
+
+on:
+  workflow_call:
+    secrets:
+      NPM_TOKEN:
+        required: true
+  workflow_run:
+    workflows: ["Integration tests"]
+    branches: [main]
+    types:
+      - completed
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Build package
+        run: yarn build
+
+      - name: Publish package
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
```

Comment:
[semantic-release](https://github.com/semantic-release/semantic-release) relies on the enforcement of specific commit messages. We don't do it atm, so deployment versioning issues may be expected... I recommend either use commitizen or just drop semantic-release for `npm publish`

Reply:
Oh actually just saw you added commitizen. So we're moving to this convention now, ok!